src/verifier/tdx/eventlog.rs

```diff
@@ -125,16 +73,13 @@ impl<'a> CcEventLog<'a> {
     pub fn query_digest(&self, entity: MeasuredEntity) -> Option<String> {
         let event_desc_prefix = Self::generate_query_key_prefix(entity)?;
 
-        for (header, data) in self.cc_events {
-            if data.len() < event_desc_prefix.len() {
+        for event_entry in self.cc_events.log.clone() {
+            if event_entry.event_desc.len() < event_desc_prefix.len() {
                 continue;
             }
-            if &data[..event_desc_prefix.len()] == event_desc_prefix.as_slice() {
-                let digest = &header.digest.digests[0];
-                if digest.hash_alg == TPML_ALG_SHA384 {
-                    let sha384 = digest.digest.sha384;
-                    return Some(hex::encode(sha384));
-                }
+            if &event_entry.event_desc[..event_desc_prefix.len()] == event_desc_prefix.as_slice() {
+                let digest = &event_entry.digests[0].digest;
+                return Some(hex::encode(digest));
```

Comment:
We can directly use `Some(..)` here without `return`

Reply:
So as above.